### ISSUE TITLE
[Bug]: `source` function calls with constant path are reported as network operations

### ISSUE THREAD

**Issue by OliverGerstl** [flowr-analysis/flowr#2524]
### What happened?

For example:
```
R> :query @linter rules:network-functions 'source("text.R")'
Query: linter (13 ms)
   ╰ **Network Functions** (network-functions):
       ╰ certain:
           ╰ Function `source` at 1.1-16
       ╰ _Metadata_: <code>totalCalls: 1, totalFunctionDefinitions: 1, searchTimeMs: 12, processTimeMs: 1</code>
All queries together required ≈13 ms (1ms accuracy, total 16 ms)
```

### What version of flowR did you use?

v2.10.6 (May 29, 2026)

### Relevant log output

```Shell

```

### Meta Information

- [ ] The issue is urgent
- [ ] I used flowR as a docker image
